### PR TITLE
update changelog for upcoming 0.7.0

### PR DESCRIPTION
**Pull request by jasonwilliams**

### CODE REVIEW

**Comment by Razican** [comment]
I think #261 could also go in this release :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+- [FEATURE #268](https://github.com/jasonwilliams/boa/pull/268):
+  Addition of forEach() (@jasonwilliams) (@xSke)
+- [FEATURE #262](https://github.com/jasonwilliams/boa/pull/262):
+  Implement Array.prototype.filter (@Nickforall)
+- [FEATURE #261](https://github.com/jasonwilliams/boa/pull/261):
+  Improved parser error messages (@Razican)
+- [FEATURE #277](https://github.com/jasonwilliams/boa/pull/277):
+  Add a logo to the project (@HalidOdat)
+- [FEATURE #260](https://github.com/jasonwilliams/boa/pull/260):
+  Add methods with f64 std equivelant to Math object (@Nickforall)
```
Comment (Razican) [inline CHANGELOG.md]:
```suggestion
  Add methods with f64 std equivalent to Math object (@Nickforall)
```

**Comment by jasonwilliams** [comment]
@Razican @HalidOdat updates were made here

**Comment by Razican** [comment]
The explanation about the new parser seems really nice :) this looks good for me!

**Pull request merged**